doc/glossary.rst

```diff
@@ -143,19 +143,22 @@ General Concepts
         as software regressions.
 
     categorical feature
-        A categorical or nominal :term:`feature` is one that has a finite set
-        of discrete values across the population of data.  These are commonly
-        represented as columns of integers or strings. Strings will be rejected
-        by most scikit-learn estimators, and integers will be treated as
-        ordinal or count-valued. For the use with most estimators, categorical
-        variables should be one-hot encoded.
-        :class:`~sklearn.preprocessing.CategoricalEncoder` helps encoding
-        string-valued categorical features.  Some estimators may handle
-        categorical features better when one-hot encoded.  See also
-        :ref:`preprocessing_categorical_features` and the
+        A categorical or nominal :term:`feature` is one that has a
+        finite set of discrete values across the population of data.
+        These are commonly represented as columns of integers or
+        strings. Strings will be rejected by most scikit-learn
+        estimators, and integers will be treated as ordinal or
+        count-valued. For the use with most estimators, categorical
```

Comment:
There goes the two spaces after a full stop almost consistency for good ;-). Personally I just think: "shrug".